packages/front-end/services/config.ts

```diff
@@ -27,11 +27,33 @@ export function useConfigJson({
       dimensions?: Record<string, Partial<DimensionInterface>>;
     } = {};
 
+    const orgSettingsFields: (keyof OrganizationSettings)[] = [
+      "environments",
+      "attributeSchema",
+      "namespaces",
+      "metricAnalysisDays",
+      "northStar",
+      "updateSchedule",
+      "multipleExposureMinPercent",
+      "videoInstructionsViewed",
+      "sdkInstructionsViewed",
+      "defaultRole",
+      "metricDefaults",
+    ];
+
+    const orgSettings = {
+      pastExperimentsMinLength: settings.pastExperimentsMinLength ?? 6,
+      visualEditorEnabled: !!settings.visualEditorEnabled,
```

Comment:
No need to define these inline anymore.  You can just add `pastExperimentsMinLength` and `visualEditorEnabled` to the fields array instead.  Don't worry about the default values or null fallbacks, they aren't important anymore.